src/main/java/org/folio/services/item/ItemService.java

```diff
@@ -176,4 +185,27 @@ public ItemService(Context vertxContext, Map<String, String> okapiHeaders) {
   private Function<SQLConnection, Future<RowSet<Row>>> updateSingleItemBatchFactory(Item item) {
     return connection -> itemRepository.update(connection, item.getId(), item);
   }
+
+  private Future<Item> refuseWhenStatisticalCodeIdDoesNotExist(final Item item) {
+    if (item.getStatisticalCodeIds().isEmpty()) {
```

Comment:
Can `item.getStatisticalCodeIds()` be null because statisticalCodeIds is not a required field?
https://github.com/folio-org/mod-inventory-storage/blob/v22.0.2/ramls/item.json#L428

Reply:
I suppose it could. Could add a null check. I think the generate model instantiates the array but probably shouldn't rely on that.